test/correctness/simd_op_check.cpp

```diff
@@ -1937,6 +1937,10 @@ int main(int argc, char **argv) {
         test.set_num_threads(1);
     }
 
+    if (getenv("FILTER")) {
```

Comment:
IMHO a less-generic env var name would be preferable here.

Reply:
Never meant to commit this, reverting

Reply:
Except you didn't. At least call is SIMD_OP_CHECK_FILTER or something. FILTER is generic enough to already be in someone's environment